File: Backend/src/main/java/com/ecommerce/app/dao/UsersResp.java
package com.ecommerce.app.dao;

import com.ecommerce.app.entity.User;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.List;

@Data
@NoArgsConstructor
public class UsersResp {

    private List<User> content;
    private int pageNumber;
    private int pageSize;
    private long totalElements;
    private int totalPages;
    private boolean lastPage;
}

File: Frontend/src/app/admin/UsersResp.ts
import { UserModel } from "../shared/model/user.model";
import { User1Model } from "../shared/model/user1.model";

export interface UsersResp {
    content: User1Model[];
    pageNumber: number;
    pageSize: number;
    totalElements: number;
    totalPages: number;
    lastPage: boolean;
  }
  